tests/unit/flows/not/flow4.yml

```diff
@@ -0,0 +1,4 @@
+jtype: Flow
+executors:
+  - name: E1
+    uses: ${{ E1_USES }}
```

Comment:
Please add more tests with `${{ ENV.E1_USES }}`, `${{ CONTEXT.E1_USES }}` variables. You can refer to all the syntax supported in [core docs](https://docs.jina.ai/get-started/migrate/#consistent-yaml-parsing-syntax).